pkg/util/util.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strings"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
+	nodeutil "k8s.io/component-helpers/node/util"
```

Comment:
Because the import lib has been updated, you need to execute `make verify` and commit the changes to update go.mod and vendor.

Reply:
Fixed. I ran the command under a directory that didn't contains the `.git` dir.